### DISCUSSION TITLE
Could not open https://mooseframework.inl.gov/in_and_out/constraint/frictional_lm.i

### DISCUSSION THREAD

**Discussion by 670262739** [idaholab/moose#25115] [Q&A Modules: Solid mechanics] [answered]
When I tried to browse https://mooseframework.inl.gov/in_and_out/constraint/frictional_lm.i, it showed 404 Not Found. Is there any way that I could be access to frictional_lm.i?

**Comment by GiudGiud** [comment, marked as answer]
Hello

Is that an input file?
You can find it on the github repo if so

Guillaume

**Reply by 670262739** [reply]
I searched with **in_and_out** and I found similar input files on the github repo.
Thank you!
